terraform/environments/mlra/locals.tf

```diff
@@ -23,7 +23,8 @@ locals {
 
   # Stores modernisation platform account id for setting up the modernisation-platform provider
   modernisation_platform_account_id = data.aws_ssm_parameter.modernisation_platform_account_id.value
-
+  #Get Pagerduty keys from modplatform
+  pagerduty_integration_keys = jsondecode(data.aws_secretsmanager_secret_version.pagerduty_integration_keys.secret_string)
```

Comment:
Let's move this over to a new file to minimise chance of conflict with other branches.

Reply:
moved to new file